truenas/webui
# NAS-112848: Refactor NTP Server Form

Changes:

- Use `app-table` component on `System Settings -> General` page for NTP Servers
- Refactor NTP Server form to new ix-forms
- Add unit tests for the new form

How to test:

- Go `System Settings -> General`
- Check the NTP Server table
- Check the NTP Server form
